applications/asset_tracker/README.rst

```diff
@@ -52,6 +52,49 @@ The `LTE Link Monitor`_ application, implemented as part of `nRF Connect for Des
 
 By default, the asset tracker supports firmware updates through :ref:`lib_aws_fota`.
 
+.. _power_opt:
+
+Power optimization
+==================
+
+The Asset Tracker can run in three power modes that are configured in the Kconfig file of the application.
+These settings are currently only supported on the nRF9160 DK.
+
+.. note::
+   Not all cellular network providers support these modes, and the granted parameters can vary between networks.
+
+Demo mode
+	This is the default setting.
+	In this mode, the device maintains a continuous cellular link.
+	To enable this mode, set ``CONFIG_POWER_OPTIMIZATION_ENABLE=n``.
+
+Request eDRX mode
+	In this mode, the device requests the eDRX feature from the cellular network to save power.
+	To enable this mode, set ``CONFIG_POWER_OPTIMIZATION_ENABLE=y`` and then
+	set Switch 2 to the N.C. position.
+
+Request Power Saving Mode (PSM)
+	In this mode, the device requests the PSM feature from the cellular network to save power.
+	To enable this mode, set ``CONFIG_POWER_OPTIMIZATION_ENABLE=y`` and then
+	set Switch 2 to the GND position.
+
+Concurrent GPS and LTE
+++++++++++++++++++++++
+
+An nRF9160-based device supports LTE-M and NB-IoT along with power saving features such as eDRX and PSM.
+The following figure shows how the data transfer occurs in an nRF9160-based device with energy conservation in place.
+
+.. figure:: /images/power_consumption.png
+   :alt: Power consumption
+
+See `Energy efficiency`_ for more information.
+
+Asset tracker enables the working of concurrent GPS and LTE in eDRX and PSM modes when the device is in `RRC idle mode`_.
+The time between the transition of a device from RRC connected mode (data transfer mode) to RRC idle mode is dependent on the network.
+Typically the time ranges between 5 seconds to 70 seconds after the last data transfer on LTE.
+
+A GPS operation occurs when the device is either not reachable (PSM) or when the device is in the sleep mode (eDRX).
```

Comment:
This is not completely correct. We state above "Asset tracker enables the working of concurrent GPS and LTE in eDRX and PSM modes when the device is in `RRC idle mode`_."
I think this linecan be removed." GPS operation occurs when the device is either not reachable (PSM) or when the device is in the sleep mode (eDRX)."

Reply:
Removed the sentence.